src/test/integration/suite/environments.test.it.ts

```diff
@@ -112,7 +112,7 @@ suite("environments", function () {
     await initTest([]);
     await waitFor(() =>
       expect(getEnvironmentViewMessage()).toBe(
-        `couldn’t find environments associated with current branch "main" Note: This view displays only environments specifically associated with the current working branch. Environments created without specifying a branch (automatically associating them with the default branch) are not displayed, even if your current branch is the default one.`
+        `couldn’t find environments associated with current branch "main" Note: This view displays only environments specifically associated with the current working branch.`
```

Comment:
Maybe we should extract those messages to a constants file to make it easier to change since I see you had to change it manually in two places

Reply:
f934e53f992969803ee62d494ed42b0be6b00b0d
124a2146d61f1add26d4c3d7c8eef629895ca274